site2/docs/deploy-monitoring.md

```diff
@@ -35,60 +34,52 @@ http://$BROKER_ADDRESS:8080/metrics
 
 ### ZooKeeper stats
 
-The local Zookeeper/configuration store server and clients that are shipped with Pulsar have been instrumented to expose
-detailed stats through Prometheus as well.
+The local Zookeeper and configuration store server and clients that are shipped with Pulsar have been instrumented to expose detailed stats through Prometheus as well.
 
 ```shell
 http://$LOCAL_ZK_SERVER:8000/metrics
 http://$GLOBAL_ZK_SERVER:8001/metrics
 ```
 
-The default port of local ZooKeeper is `8000` and that of configuration store is `8001`.
-These can be changed by specifying system property `stats_server_port`.
+The default port of local ZooKeeper is `8000` and the default port of configuration store is `8001`. You can change the default port of local Zookeeper and configuration store by specifying system property `stats_server_port`.
 
 ### BookKeeper stats
 
 For BookKeeper you can configure the stats frameworks by changing the `statsProviderClass` in
 `conf/bookkeeper.conf`.
 
-By default, the default BookKeeper configuration included with Pulsar distribution will enable
-the Prometheus exporter.
+By default, the default BookKeeper configuration included with Pulsar distribution enables the Prometheus exporter.
```

Comment:
It's technically correct, you can refine it further, and make it easier to read and understand.